src/Microsoft.DocAsCode.Dfm/CodeSnippetExtractor/RecursiveNameCodeSnippetExtractor.cs

```diff
@@ -52,6 +51,10 @@ protected override List<CodeSnippetTag> ResolveCodeSnippetTags(string[] lines)
                 var endLineMatch = _endLineRegex.Match(lines[i]);
                 if (endLineMatch.Success)
                 {
+                    if (snippetStack.Count == 0)
+                    {
+                        throw new Exception($"line {i} contains an end line that can't match a start line");
```

Comment:
Sure, I catch this exception and add a LogError in ResolveTagNamesFromPath.

---
In reply to: [110374469](https://github.com/dotnet/docfx/pull/1528#discussion_r110374469) [](ancestors = 110374469)